administration/runtime.md

```diff
@@ -133,6 +133,70 @@ Usage: openhab:send <item> <command> - sends a command for an item
 
 Apache Karaf itself also provides many useful commands, especially for debugging and development purposes.
 
+### Bundle Management
+
+Karaf provides a number of commands to manage bundles.
+
+Use the `bundle --help` command to get a list of all available bundle management commands.
+`bundle:<command> --help` provides more information about a specific command.
+
+The following commands are particularly useful:
+
+- `bundle:list`: Lists all installed bundles.
+- `bundle:list <bundleName>`: Lists a bundle by name, e.g.:
```

Comment:
How do I know the name? Even when I list all, it doesn't show the name in the table.  I see this is discussed a bit below so it might be worth mentioning here that it is discussed below. The question arises here and it's not clear the answer will be in the doc anywhere. Simply adding a "(see below for how to discover the name)" after "name" would be sufficient.